pandas/core/internals/construction.py

```diff
@@ -430,6 +430,12 @@ def _get_axes(N, K, index, columns):
     return index, columns
 
 
+def _dataclasses_to_dicts(data):
```

Comment:
can you try to type things (you can add in a TYPE_CHECKING block if need be)

Reply:
ok